### DISCUSSION TITLE
audio THEN buttons

### DISCUSSION THREAD

**Discussion by rbruno1** [jspsych/jsPsych#1943] [Q&A] [answered]
I am trying to avoid phonological representation as I present my stimuli with the audio-button_response plugin, and I realized that if I was looking at one of the labels on the button, i was more partial to hearing that word (it is a word disambiguation task) 
How do I display the buttons AFTER the audio, and not during?

**Comment by jodeleeuw** [comment, marked as answer]
The simplest way is to use the audio-keyboard-response plugin with no response allowed to play the audio, and then a separate button trial to collect the response.
